README.md

```diff
@@ -1,41 +1,420 @@
+
+## Key Features of MongoDB
+
+- List database names
+- List collection names
+- Insert
+- Update
+- Find
+- Delete
+- Count
+
+## Connector Overview
+
+The Mongo DB connector allows you to connect to a Mongo DB from Ballerina and perform various operations such as `getDatabaseNames`, `getCollectionNames`, `count`, `listIndices`, `find`, `insert`, `update`, and `delete`.
+
+# Prerequisites
+
+* A mongodb with username and password
+
+* Java 11 Installed <br/> Java Development Kit (JDK) with version 11 is required.
+
+* Ballerina SLAlpha4 Installed <br/> Ballerina Swan Lake Alpha 4 is required.
+
+# Supported Versions 
 
 |                             |       Version               |
 |:---------------------------:|:---------------------------:|
-| Ballerina Language          | Swan Lake Alpha 2           |
+| Ballerina Language          | Swan Lake Alpha 4           |
 | Mongo DB                    | V4.2.0                      |
 
-## MongoDB Clients
 
-There are 3 clients provided by Ballerina to interact with MongoDB.
 
-1. **mongodb:Client** - This connects to the running MongoDB node and lists the database names as well as gets a client for a specific database.
+# Quickstart(s)
+
+## Insert a document
+
+### Step 1: Import the Mongo DB module
+First, import the `ballerinax/mongodb` module into the Ballerina project.
+```ballerina
+import ballerinax/mongodb;
+```
+### Step 2: Set up configurable values
+You can add required variables as configurable values in the ballerina file and can add those values in `Config.toml` file. 
+1. In Ballerina file 
+```ballerina
+configurable string host = ?;
+configurable int port = ?;
+configurable string username = ?;
+configurable string password = ?;
+configurable string database = ?;
+configurable string collection = ?;
+```
+2. In Config.toml
+
+```
+host = "<YOUR_HOST_NAME>""
+port = <PORT>
+username = "<DB_USERNAME>"
+password = "<DB_PASSWORD>"
+
+database = "<DATABASE_NAME>"
+collection = "<COLLECTION_NAME>"
+```
+
+### Step 3: Initialize the Mongodb Client giving necessary credentials
+
+You can now enter the credentials in the mongo client config. If you use this client for a particular database then you can pass the database name along with config during client initialization(It is optional). Otherwise you can pass the database name for each remote method call. This is not recommended unless you need to connect more than one database using a client. You need to set the database using atleast one of these methods.
+```ballerina
+mongodb:ClientConfig mongoConfig = {
+        host: host,
+        port: port,
+        username: username,
+        password: password,
+        options: {sslEnabled: false, serverSelectionTimeout: 5000}
+    };
+
+    mongodb:Client mongoClient = checkpanic new (mongoConfig, database);
```

Comment:
Please provide commit or PR if `checkpanic` already has been fixed in the master?

Reply:
This code snippet is now removed from here and added in Module.md file. This change is done via the following PR https://github.com/ballerina-platform/module-ballerinax-mongodb/pull/111 . You can see the change in Module.md https://github.com/ballerina-platform/module-ballerinax-mongodb/pull/111/files#diff-db8c92970e0763b610645268a9105ee93f96ad2fcae5c68d92d348634b27aa77R38 here.